src/Adapter/Module/Tab/ModuleTabRegister.php

```diff
@@ -275,15 +282,18 @@ protected function registerTab(Module $module, ParameterBag $tabDetails)
     {
         $this->checkIsValid($module->get('name'), $tabDetails);
 
-        /**
-         * Legacy Tab, to be replaced with Doctrine entity when right management
-         * won't be directly linked to the tab creation
-         *
-         * @ToDo
-         */
+        // Legacy Tab, to be replaced with Doctrine entity when right management
+        // won't be directly linked to the tab creation
+        // @ToDo
+
+        $className = $tabDetails->get('class_name');
+
+        if (is_subclass_of($className, FrameworkBundleAdminController::class)) {
```

Comment:
What's the difference between this and

```
         if ($className instanceof FrameworkBundleAdminController) { 
```

Reply:
An example si better than a long story: https://www.php.net/manual/en/internals2.opcodes.instanceof.php#109108
But don't know why you use `is_subclass_of` here :thinking:

Reply:
Yeah I know the difference, my question was actually the same as yours. Is there a specific need here?

Reply:
if I remember well, `instanceof` only accepts instances and not class names: if I'm wrong, I'm also in favor of using `instanceof` instead.

> [instanceof](https://www.php.net/manual/fr/language.operators.type.php) est utilisé pour déterminer si une variable PHP est un objet instancié d'une certaine classe